pkg/didcomm/dispatcher/api.go

```diff
@@ -22,6 +22,24 @@ type Service interface {
 type MessageService interface {
 	service.InboundHandler
 	Accept(header *service.Header) bool
+	Name() string
+}
+
+// MessageServiceProvider is provider of message services
+type MessageServiceProvider interface {
```

Comment:
This is really part of the framework rather than the dispatcher package.

Reply:
moved to framework api location.